### DISCUSSION TITLE
Hover styling with drawer

### DISCUSSION THREAD

**Discussion by elliott-farrall** [Alexays/Waybar#3157] [Q&A]
Has anyone been able to get drawers working since #3139? I am now able to have styles for modules using :hover but any drawers I create will not open when I hover over them. Not sure if an issue needs to be raised or if I am just missing something.

Edit: Created an issue (#3162).

**Comment by SnakeJaeger** [comment]
I can confirm drawers are not functioning for me at the moment either.

**Reply by elliott-farrall** [reply]
Thanks for checking this. I will look into creating an issue.
